packages/components/nodes/vectorstores/ZepCloud/ZepCloud.ts

```diff
@@ -0,0 +1,231 @@
+import { flatten } from 'lodash'
+import { IDocument, ZepClient } from '@getzep/zep-cloud'
+import { IZepConfig, ZepVectorStore } from '@getzep/zep-cloud/langchain'
+import { Embeddings } from 'langchain/embeddings/base'
+import { Document } from 'langchain/document'
+import { ICommonObject, INode, INodeData, INodeOutputsValue, INodeParams } from '../../../src/Interface'
+import { getBaseClasses, getCredentialData, getCredentialParam } from '../../../src/utils'
+import { addMMRInputParams, resolveVectorStoreOrRetriever } from '../VectorStoreUtils'
+import { FakeEmbeddings } from 'langchain/embeddings/fake'
+class Zep_CloudVectorStores implements INode {
+    label: string
+    name: string
+    version: number
+    description: string
+    type: string
+    icon: string
+    category: string
+    badge: string
+    baseClasses: string[]
+    inputs: INodeParams[]
+    credential: INodeParams
+    outputs: INodeOutputsValue[]
+
+    constructor() {
+        this.label = 'Zep Collection - Cloud'
+        this.name = 'Zep Collection - Cloud'
```

Comment:
the `name` must be in camelcase `zepCloud`, otherwise the upsert button will not be shown: 
<img width="919" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/26460777/302532c6-679d-4e5b-9be7-cbab66cc055a">